centos_ci.sh

```diff
@@ -292,14 +292,40 @@ function perform_release() {
   curl http://minibot.19cf262c.svc.dockerapp.io:9009/hubot/centosci -H "Content-Type: application/json" -d '{"payload":{"status":"success","message":'"\"$MESSAGE\""',"url":'"\"$URL\""'}}' || true
 }
 
+function perform_docs_publish() {
+  # REPO and BRANCH variables are populated via https://ci.centos.org/job/minishift-docs
+  DOC_USER=$(echo $REPO | cut -d"/" -f4)
+  cd gopath/src/github.com/minishift/minishift
+
+  # Fetch remote repo and checkout to specified branch if not default minishift repo
+  if [[ "$DOC_USER" != "minishift" ]]; then
+    git remote add remote-user $REPO
```

Comment:
remote-user does not seem to be the right name either. Remote repo or custom-repo or custom-remote. You are adding a remote repository not a user.